### DISCUSSION TITLE
opendtu wrong values on screen

### DISCUSSION THREAD

**Discussion by E-Bzh49** [tbnobody/OpenDTU#2425] [General]
Hello everybody. I'am a new french user of an homemade opendtu. (sorry for my english).
My Hardware is : ESP32 Wroom + screen SH1106 + RF module CMT2300A.
My soft version is 24.10.15.
I have 3 hoymiles inverter HMS 1000.

All is working well but I have 2 différents value between the openDTU screen and the hoymiles DTU pro app.
For exemple when hoymiles app tells 138W of solar product the open dtu shows 68w.
The values were compared at the refesh time of the app. I can see all the inverters connected on the web page.

Have you got any ideas, If I delete one and after 2 inverters, the displayed value of solar product decreases.

Thanks in advance for any help.

**Comment by stefan123t** [comment]
@E-Bzh49 bienvenue chez OpenDTU-OnBattery.
Have you checkes the settings for the display. I think there may be a selection for which of the inverters to diaplay the values or cycle between invertera and/or total values.

@schlimmchen do we update the display and live view at the same time ?
If we are collecting data from the inverters between those updates of the live view and display some difference may be noticable.

**Reply by E-Bzh49** [reply]
Thanks for your help. I think I have found the reason. When I start my openDTU that makes fall down 2 of my inverters. I needed to call my reseller. After that all was working I try to restart my openDTU and again 2 inverters stop their solar product.

**Comment by E-Bzh49** [comment]
Hi I answer to myself. I understand my problem. When I have openDTU and dtu pro hoymiles together a conflict appears between them and no datas are received on any receivers.
I read on older comments that some people were be able to have both systems together, is it an issue due to a new version of openDTU.
